src/main/java/be/syntra/devshop/DevshopFront/controllers/ProductController.java

```diff
@@ -103,12 +104,35 @@ public String addSelectedProductFromDetailToCart(@PathVariable Long id) {
         return "redirect:/products/details/" + id;
     }
 
+    @PostMapping("/details/{id}/addreview")
```

Comment:
preferable to use underscore: e.g. add_review. Same goes for the other mappings

Reply:
fixed